Formula/curl.rb

```diff
@@ -52,6 +52,8 @@ def install
     system "make", "install"
     system "make", "install", "-C", "scripts"
     libexec.install "lib/mk-ca-bundle.pl"
+    
+    zsh_completion.install  "/usr/local/opt/curl/share/zsh/site-functions"  => "_curl"
```

Comment:
Not every install is `/usr/local`. Please read https://docs.brew.sh/Formula-Cookbook#just-moving-some-files

Reply:
fixed it